SPECS/tokyocabinet/tokyocabinet.spec

```diff
@@ -1,52 +1,57 @@
-Summary:	A modern implementation of a DBM
-Name:		tokyocabinet
-Version:	1.4.48
-Release:	16%{?dist}
-License:	LGPLv2+
+Summary:        A modern implementation of a DBM
+Name:           tokyocabinet
+Version:        1.4.48
+Release:        17%{?dist}
+License:        LGPLv2+
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
-URL:		http://fallabs.com/tokyocabinet/
-Source:		http://fallabs.com/%{name}/%{name}-%{version}.tar.gz
-Patch0:		tokyocabinet-fedora.patch
-Patch1:		tokyocabinet-manhelp.patch
-BuildRequires:	pkgconfig zlib-devel bzip2-devel autoconf gcc
+URL:            http://fallabs.com/tokyocabinet/
+Source:         https://github.com/maiha/tokyocabinet/archive/refs/heads/v48.tar.gz
+#Source:         http://fallabs.com/%{name}/%{name}-%{version}.tar.gz
```

Comment:
The GitHub repo says it's an unofficial repo, so let's use the official one instead. I think the original source URL was fine.

Reply:
The original link doesn't pass the tests, so not sure what other links to use.

Reply:
It passed for me locally - maybe it was a one-time thing. Try again after applying the changes I've mentioned in `malaga.spec`.